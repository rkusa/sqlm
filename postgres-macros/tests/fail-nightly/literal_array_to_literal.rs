../fail-stable/literal_array_to_literal.rs