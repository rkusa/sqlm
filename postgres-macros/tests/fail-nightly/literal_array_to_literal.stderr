error[E0277]: the trait bound `String: sqlm_postgres::query::Query<sqlm_postgres::types::Array<Vec<String>>>` is not satisfied
 --> tests/fail-nightly/literal_array_to_literal.rs:6:10
  |
6 |         .await
  |         -^^^^^
  |         ||
  |         |the trait `sqlm_postgres::query::Query<sqlm_postgres::types::Array<Vec<String>>>` is not implemented for `String`
  |         help: remove the `.await`
  |
  = help: the following other types implement trait `sqlm_postgres::query::Query<Cols>`:
            `()` implements `sqlm_postgres::query::Query<()>`
            `Option<T>` implements `sqlm_postgres::query::Query<Primitive<<T as SqlType>::Type>>`
            `Option<T>` implements `sqlm_postgres::query::Query<Struct<Cols>>`
            `Option<Vec<u8>>` implements `sqlm_postgres::query::Query<Primitive<Bytea>>`
            `Vec<T>` implements `sqlm_postgres::query::Query<Primitive<<T as SqlType>::Type>>`
            `Vec<T>` implements `sqlm_postgres::query::Query<Struct<Cols>>`
            `Vec<T>` implements `sqlm_postgres::query::Query<sqlm_postgres::types::Array<Vec<<T as SqlType>::Type>>>`
            `Vec<Vec<u8>>` implements `sqlm_postgres::query::Query<Primitive<Bytea>>`
          and $N others
  = note: required for `Sql<'_, sqlm_postgres::types::Array<Vec<String>>, String>` to implement `IntoFuture`
